CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # CHANGELOG
 
+## 2.0.0 (2022-11-14)
+
+MLflow 2.0.0 includes several breaking changes from the 1.x API, additional major features and improvements
+
+Breaking Changes:
+
+- [Recipes] Renamed `mlflow.pipelines` to `mlflow.recipes`
```

Comment:
```suggestion
- [Recipes] Renamed MLflow Pipelines to MLflow Recipes
```

Reply:
Can we introduce a separate breaking change item indicating that the `mlflow.pipelines` APIs have been replaced with `mlflow.recipes` APIs?